Exonoesis/Helping-Hand
# Audio settings menu

## What is the purpose of these changes?
To add the ability to modify the volume of the music and sound effects within the game as well as serve as a place to modify other aspects of the game in the future.

## What has changed?
There is now a button on the Main Menu which opens a new menu called Settings. The Settings Menu contains sliders and spinners which, when manipulated, alter the volumes of the music and sound effects within the games play state.

## How can I verify this works?
1. Launch the game
2. Click the button labeled Settings
3. Here you can change the slider values via grabbing the tan square handles and moving them to the correct spot (good for large movements) as well as by clicking the up and down arrows next to the displayed value (good for fine tuning).
4. Make note of the values you set and click apply
5. You will be returned to the Main Menu. Click the Play button to enter the game
6. Note that the games music will be at the level you set the volume slider, while the sound made when the player walks and bumps into objects will be at the level of the SFX (sound effect) slider
7. Feel free to close the game and repeat steps 1-6 with different volumes until satisfied 

---

Closes #40
